### PR TITLE
Feature #67 Game Center login for Profile

### PR DESCRIPTION
**Pull request by EugeneButusov**

### CODE REVIEW

File: SoomlaiOSProfile/UserProfileUtils.h [hunk 1]
```diff
@@ -22,7 +22,8 @@
 typedef enum {
     FACEBOOK = 0,
     GOOGLE = 2,
-    TWITTER = 5
+    TWITTER = 5,
+    GAMECENTER = 7
```
Comment (vedi) [inline SoomlaiOSProfile/UserProfileUtils.h]: @EugeneButusov, maybe `GAME_CENTER`? Why 7?

File: SoomlaiOSProfile/UserProfileUtils.m [hunk 2]
```diff
@@ -31,6 +31,8 @@ + (NSString *)providerEnumToString:(Provider)provider {
             return @"google";
         case TWITTER:
             return @"twitter";
+        case GAMECENTER:
+            return @"gamecenter";
```
Comment (vedi) [inline SoomlaiOSProfile/UserProfileUtils.m]: @EugeneButusov, let's camelCase it here

File: SoomlaiOSProfile/UserProfileUtils.h (hunk 1 above)
Comment (borisspektor) [inline SoomlaiOSProfile/UserProfileUtils.h]: @EugeneButusov @vedi make it 13 (we had other providers until 11, and 12 is currently saved for someone)

File: SoomlaiOSProfile/UserProfileUtils.m (hunk 2 above)
Comment (EugeneButusov) [inline SoomlaiOSProfile/UserProfileUtils.m]: @vedi, ok, sure.

File: SoomlaiOSProfile/UserProfileUtils.h (hunk 1 above)
Comment (vedi) [inline SoomlaiOSProfile/UserProfileUtils.h]: @borisspektor, what's "other providers"? I never heard this.

Comment (borisspektor) [inline SoomlaiOSProfile/UserProfileUtils.h]: @vedi If you look in the history of this file you'll see what I'm talking about

**Pull request merged**